Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 55;
+	objectVersion = 70;
```

Comment:
Please revert this version change.

Reply:
Oops this didn't get fixed in this second commit. Let me re-commit with this change.

Reply:
Actually the project won't build right now because I accidentally didn't add the pbxproj to the commit. Apologies. Let me re-generate & re-add.